modules/ROOT/pages/functions/temporal/duration.adoc

```diff
@@ -346,6 +345,7 @@ duration.inMonths(instant1, instant2)
 | If `instant1` has a time zone component and `instant2` does not, the time zone component of `instant2` is assumed to be the same as that of `instant1`, and vice versa.
 | If `instant1` has a date component and `instant2` does not, the date component of `instant2` is assumed to be the same as that of `instant1`, and vice versa.
 | Any difference smaller than a whole month is disregarded.
+| To get the actual number of months as an `INTEGER`, call the months component of the returned `DURATION` object via `.months`, see see xref:values-and-types/temporal.adoc#cypher-temporal-accessing-components-durations[Components of durations].
```

Comment:
```suggestion
| You can get the duration's total number of months through the `.months` component (see xref:values-and-types/temporal.adoc#cypher-temporal-accessing-components-durations[Components of durations]).
```
_Component_ is a misleading word everywhere in these pages, as it implies it is only _part_ of the duration.

_Actual_ is also questionable, as it implies it is a more _real_ view than the duration itself. It's not: it's just a different representation.

These apply to further instances below, for seconds, minutes, etc.